etc/helm/pachyderm/templates/pachd/config-secret.yaml

```diff
@@ -24,26 +24,28 @@ stringData:
   # authConfig is the configuration for pachd as an OIDC client
   # ** note: `authConfig.client_secret` must equal `oidcClients.secret`
   authConfig: |
-    client_id: {{ required "For Authentication, an OIDC Client ID for this pachd must be set." .Values.pachd.oauthClientId }}
+    client_id: {{ required "For Authentication, an OIDC Client ID for this pachd must be set." .Values.pachd.oauthClientID }}
     client_secret: {{ $oidcSecret }}
-    issuer: {{ required "For Authentication, an OIDC Issuer for this pachd must be set." .Values.pachd.oauthIssuer }}
-    localhost_issuer: true
-    require_email_verified: false
-    user_accessible_issuer_host: {{ .Values.pachd.userAccessibleOauthIssuerHost }}
-    redirect_uri: {{ required "For Authentication, an OIDC Redirect URI for this pachd must be set." .Values.pachd.oauthRedirectURI }}
+    issuer: {{ include "pachyderm.issuerURI" . }}
+    localhost_issuer: {{ include "pachyderm.localhostIssuer" . }}
+    require_email_verified: {{ .Values.oidc.requireVerifiedEmail }}
+    {{- if include "pachyderm.userAccessibleOauthIssuerHost" . }}
+    user_accessible_issuer_host: {{ include "pachyderm.userAccessibleOauthIssuerHost" . }}
+    {{- end }}
+    redirect_uri: {{ include "pachyderm.pachdRedirectURI" . }}
     scopes:
     - email
     - profile
     - groups
     - openid 
 
-{{ if eq .Values.deployTarget "LOCAL" }}
+  # TODO: When should this _not_ be specified?
```

Comment:
I think I can remove this entirely. It's supposed to serve as an example for pushing pachyderm Cluster Role bindings to pachd

Reply:
yeah sure. How does having this piece in a comment block sound?